### ISSUE TITLE
WatchCartoonOnline is down

### ISSUE THREAD

**Issue by bjgood** [spoyser/spoyser-repo#60]
Hi Sean,
Hope you check here at your github soon. :).

Anyways, WatchCartoonOnline is down. They've made some changes on the website, obviously, and WCO is no longer working. When you get the chance could you please take a look at it?

I'm Trespasser from the Kodi forum. I sent you a PM but you haven't visited there in like forever. :D. The kids and I are getting desperate without WCO to watch.

Sorry for trying to contact you here, bud.

Best wishes,
Bob

**Comment by kodibear** [comment]
I don't think this repository is maintained anymore. If you have no other options, check out my fork of the plugin, which works with the changed version of WCO: [https://github.com/kodibear/plugin.video.watchcartoononline](https://github.com/kodibear/plugin.video.watchcartoononline)

**Comment by daniel-ntl** [comment]
to all here is the working WCO, works in kodi frodo and jarvis, thx to kodibear,
https://github.com/kodibear/plugin.video.watchcartoononline,
case closed
